src/commands/env/env-list.js

```diff
@@ -89,6 +122,16 @@ const envList = async (options, command) => {
 const createEnvListCommand = (program) =>
   program
     .command('env:list')
+    .addOption(
+      new Option('-c, --context <context>', 'Specify a deploy context')
```

Comment:
Similar questions for the context and scope flags here as I left for the get command